src/Table.js

```diff
@@ -344,6 +350,7 @@ Table.defaultProps = {
   className: '',
   disableHeader: false,
   emptyTablePlaceholder: null,
+  getRowKey: () => null,
```

Comment:
could maybe make this `({ rowIndex }) => 'table_row_${rowIndex}'` to keep the old behavior when not supplying a `getRowKey` prop (see https://github.com/mroyce/tangelo/pull/36/files#diff-969af7bd600b29e79fa6137a12b14ac4L152)